src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDeconstructTests.cs

```diff
@@ -4005,17 +3997,17 @@ static void Main()
 @"{
   // Code size       70 (0x46)
   .maxstack  2
-  .locals init (System.Collections.Generic.IEnumerator<(int, int)> V_0,
+  .locals init (System.Collections.Generic.IEnumerator<System.ValueTuple<int, int>> V_0,
```

Comment:
><System.ValueTuple<int, int> [](start = 54, length = 28)

Why did the representation in the IL visualizer change?

Reply:
IL visualization uses a mix of tuple syntax and `System.ValueTuple<...>`. That caused some trouble because we no longer have distinct symbols, so we must pick one representation or another. The `System.ValueTuple<...>` representation seems more suitable for IL.

---
In reply to: [355734721](https://github.com/dotnet/roslyn/pull/39370#discussion_r355734721) [](ancestors = 355734721)